### PR TITLE
Switch from downloadcontent.opensuse.org to download.opensuse.org for fetching podman and skopeo

### PR DESCRIPTION
**Pull request by iain-macdonald**

### CODE REVIEW

File: dockerfiles/executor_image/Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add - && \
 RUN apt-get update && apt-get install -y umoci iproute2
 
 # Install podman and skopeo
-RUN echo 'deb https://downloadcontent.opensuse.org/repositories/home:/alvistack/Debian_11/ /' >> /etc/apt/sources.list.d/home:alvistack.list && \
+RUN echo 'deb https://download.opensuse.org/repositories/home:/alvistack/Debian_11/ /' >> /etc/apt/sources.list.d/home:alvistack.list && \
```
Comment (luluz66) [inline dockerfiles/executor_image/Dockerfile:16]: I think we need update podman version as well
E: Version '100:4.3.0-1' for 'podman' was not found   
E: Version '100:1.10.0-1' for 'skopeo' was not found
podman 4.3 is no longer there
Screenshot from 2023-04-17 09-33-53

Comment (luluz66) [reply dockerfiles/executor_image/Dockerfile:16]: Changing to "podman=100:4.5.0-1 skopeo=100:1.12.0-1" will make docker build succeeds. However, I haven't tested executors with new versions od podman and skopeo.

**Review by luluz66 — APPROVED** [review]

**Review by bduffany — APPROVED** [review]

File: dockerfiles/executor_image/Dockerfile (hunk 1 above)
Comment (bduffany) [reply dockerfiles/executor_image/Dockerfile:16]: upgrading the versions sgtm, we can run a few TF builds and make sure nothing breaks

Comment (iain-macdonald) [reply dockerfiles/executor_image/Dockerfile:16]: Tested with a tensorflow build and it seems to work fine: https://app.buildbuddy.dev/invocation/13e14cdc-8d18-4387-9a19-e8383b678e82

**Pull request merged**